### ISSUE TITLE
Proposal discussion auto-scroll distraction

### ISSUE THREAD

**Issue by traumschule** [Joystream/pioneer#3968]
Triaging #3967 i noticed an auto-scroll behavior that keeps pulling back to the post every few seconds after clicking the 'replies to' link and scrolling further.

It should be fine to scroll only once.

**Comment by dmtrjsg** [comment]
@traumschule please feel free to assign, looks not very serious but I can see it can be very annoying 😁

**Comment by thesan** [comment]
Proposal discussions also render the `PostListItem` component. So this might be the same problem causing #4129

**Comment by chrlschwb** [comment]
@Honest0, please fix this together with #4129

**Comment by Honest0** [comment]
Am here

**Comment by traumschule** [comment]
duplicate of #4129
